tests/utils/ExampleParentTestCase.cfc

```diff
@@ -0,0 +1,20 @@
+component extends="testbox.system.BaseSpec" {
+
+    /**
+     * @beforeEach
+     */
+    function runThisBefore() {
+        variables.counter++;
+    }
+
+    /**
+     * @afterEach
```

Comment:
Does it get picked up if I have more than one of these methods annotated with `afterEach` etc?